File: baekjoon_all/02000+/boj_2525_solution2.cpp
// Solve 2022-06-02
// Update 2023-10-09

#include <iostream>
using namespace std;

int main() {
    int hour, minute, extra;
    cin >> hour >> minute >> extra;

    minute += extra;

    if (minute >= 60) {
        hour += minute / 60;
        minute %= 60;
    }

    if (hour >= 24) {
        hour %= 24;
    }

    cout << hour << ' ' << minute << '\n';

    return 0;
}

File: baekjoon_all/02000+/boj_2525_solution2.py
# Solve 2022-05-16
# Update 2023-10-09

hour, minute = map(int, input().split())
extra = int(input())

minute += extra

if minute >= 60:
    hour += minute // 60
    minute %= 60

if hour >= 24:
    hour %= 24

print(hour, minute)
